server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -449,6 +449,26 @@ public void consumerExists(
         }
     }
 
+    @ApiOperation(
+        notes = "Checks for the existence of a Consumer in bulk. This API returns uuids of non-existing" +
+        "consumer",
+        value = "")
+    @ApiResponses({ @ApiResponse(code = 400, message = "Return 404 when empty or null array is passed"),
+        @ApiResponse(code = 200, message = "Return array of UUIDs of consumers that doesn't exist or " +
+        "cannot be accessed") })
+    @POST
+    @Consumes(MediaType.WILDCARD)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("exists")
+    public List<String> consumerExistsBulk(ArrayList<String> consumerUuid) throws BadRequestException {
+        if (!CollectionUtils.isEmpty(consumerUuid)) {
+            return consumerCurator.getNonExistingConsumer(consumerUuid);
+        }
+        else {
+            throw new BadRequestException(i18n.tr("No Uuid's provided. Unable to process request."));
```

Comment:
Grammar: 
- UUID is an acronym and should be entirely capitalized (or not capitalized at all depending on context and general usage throughout the application)
- An apostrophe signifies ownership or character omission; it is not applicable when pluralizing
- Generally follow the pattern of "Error: reason" or "event; details"; e.g. "Unable to process request: no UUIDs provided". However, in this case, the error being output to the end user will already imply that it was unable to process the request (that's what 4xx and 5xx codes are for), so in some contexts it's not necessary to include that part of the message.

Reply:
Modifying it to "No UUIDs provided" as it is apparent from Bad Request Error code, that request cant be processed.